torch/csrc/autograd/profiler.cpp

```diff
@@ -516,14 +517,34 @@ void Event::record(bool record_cuda) {
       NUM_EVENT_IVALUE_IDX,
       " elements to reconstruct Event.");
 
+  // Reconstruct input shapes from ivalues.
+  auto shapeListFromServer = ivalues.get(EventIValueIdx::SHAPES);
```

Comment:
nit: could you remove mentions of server?

Reply:
Removed all of the mentions.